src/core/utils.ts

```diff
@@ -130,7 +142,7 @@ export function getPointColor(point?: Highcharts.Point): string {
 // Highcharts legend is disabled. Instead, we use this custom method to collect legend items from the chart.
 export function getChartLegendItems(chart: Highcharts.Chart): readonly LegendItemSpec[] {
```

Comment:
From the demo it looks like the colours calculation for chart legend does not produce the expected colours. What is yellow in the legend is red in the chart.